### ISSUE TITLE
Erroring out when "tns build android" is used

### ISSUE THREAD

**Issue by shivapersad** [NativeScript/NativeScript#3558]
When I run the "tns build android" command I get the following output and eventual error. I am presently running Nativescript 2.5.0, npm 4.0.2. Any assistance is welcome.

Running full build
finished with reading lines with js files
:asbg:generateBindings
:mergeF0F1F2DebugShaders
:compileF0F1F2DebugShaders
:generateF0F1F2DebugAssets
:mergeF0F1F2DebugAssets
:generateF0F1F2DebugResValues
:generateF0F1F2DebugResources
:mergeF0F1F2DebugResources
:processF0F1F2DebugManifest
:processF0F1F2DebugResources
/Users/shivapersad/cngProject/cngApp2/platforms/android/build/intermediates/res/merged/F0F1F2/debug/values-v24/values-v24.xml:3: AAPT: Error retrieving parent for item: No resource found that matches the given name 'android:TextAppearance.Material.Widget.Button.Borderless.Colored'.
    
/Users/shivapersad/cngProject/cngApp2/platforms/android/build/intermediates/res/merged/F0F1F2/debug/values-v24/values-v24.xml:4: AAPT: Error retrieving parent for item: No resource found that matches the given name 'android:TextAppearance.Material.Widget.Button.Colored'.
    
:processF0F1F2DebugResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processF0F1F2DebugResources'.
> com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command '/Users/shivapersad/Library/Android/sdk/build-tools/25.0.2/aapt'' finished with non-zero exit value 1

**Comment by tsonevn** [comment]
Hi @shivapersad,

For your case, you could try to remove `node_modules`, `platforms` and `hooks` folders, to clear npm cache with `npm cache clear`. Then try to rebuild the project.

If this does not help, please open your `Android SDK Manager` and verify, whether you have installed `Android SDK Build-tools` versions `25.0.2` and `23.0.2`. If not, install them, remove again `platforms` folder and rebuild your project.

Hope this information helps.

**Comment by shivapersad** [comment]
I was able to resolve the issue. I had to uninstall Nativescript, node, npm and Android Studio. I reinstalled Android Studio first and then node, npm and Nativescript. I didn't allow the tns setup script to run. I set the $ANDROID_HOME path variable manually in Mac OS. It seems that when I ran "tns doctor" previously it modified my $ANDROID_HOME path and even though I tried changing it manually it wouldn't change until I removed Nativescript, node and npm. I've gotten my application building in Android again thankfully.
